core/domain/auth_domain.py

```
@@ -37,10 +37,10 @@
 )
 
 
-# The minimum fields needed to link an Oppia user with authentication providers.
-ExternalAccount = collections.namedtuple(
-    'ExternalAccount', ['auth_id', 'email', 'disabled']
-)
+class AuthProviderError(Exception):
+    """Error raised when the authentication provider raises an error."""
+
+    pass
```

Ruff (0.14.14)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)